web/cobrands/sass/_mixins.scss

```diff
@@ -24,7 +24,6 @@ $button-margin-left: 0 !default;
 
 
 @mixin button-size($button-margin-top:$button-margin-top, $button-margin-right:$button-margin-right, $button-margin-bottom:$button-margin-bottom, $button-margin-left:$button-margin-left, $button-padding-top:$button-padding-top, $button-padding-right:$button-padding-right, $button-padding-bottom:$button-padding-bottom, $button-padding-left:$button-padding-left) {
-    display: inline-block !important; //Override inline display: none on #geolocation_link
```

Comment:
The "important" was added earlier in this PR, and presumably not needed, but now the whole row is being removed? Can I just check, does this need to be inline-block or not? It's been inline-block since at least 2016.

Reply:
I think I removed it in error, I'll add it back in now.

Thanks